src/internal/model/SourceID.ts

```diff
@@ -0,0 +1,11 @@
+export const SourceID = {
+  UNKNOWN: 0,
+  ANDROID: 1,
+  IOS: 2,
+  WEB: 3,
+  GOAL_BATCH: 4,
+  GO_SERVER: 5,
+  NODE_SERVER: 6,
```

Comment:
@cre8ivejp Is it ok to assume that SourceID is still a number type?

Reply:
Yes.
https://github.com/bucketeer-io/bucketeer/blob/main/proto/event/client/event.proto#L61-L69